forte/data/ontology/configs/example_ontology_config.json

```diff
@@ -7,7 +7,7 @@
     "definitions": [
         {
             "entry_name": "example_ontology.Word",
-            "parent_type": "example_import_ontology.Token",
+            "parent_entry": "example_import_ontology.Token",
```

Comment:
I have a question here. The parent entry reference did not include the prefix of the example_import_ontology (i.e. ``ft.onto``). How do we know that the full name of this ``parent_entry`` is ``ft.onto.example_import_ontology.Token``, what if the user creates another type with a different prefix, such as ``edu.cmu.example_import_ontology.Token``? Can the current semantics distinguish these?

Reply:
Changed as discussed. All the entries and types have full names. If additional names are to be defined, it is done through `additional_prefixes` field at the top level of ontology. See `forte/data/ontology/configs/example_multi_module_ontology_config.json` for an example of multi-module ontology.